awscli/customizations/cloudformation/artifact_exporter.py

```diff
@@ -199,12 +200,20 @@ def mktempfile():
             os.remove(filename)
 
 
+def copy_to_temp_dir(filepath):
+    tmp_dir = tempfile.mkdtemp()
+    dst = os.path.join(tmp_dir, os.path.basename(filepath))
+    shutil.copyfile(filepath, dst)
+    return tmp_dir
+ 
+
 class Resource(object):
     """
     Base class representing a CloudFormation resource that can be exported
     """
 
     PROPERTY_NAME = None
+    FORCE_ZIP = False
```

Comment:
I think this should be true by default, otherwise the fix doesn't work by default.